tests/e2e/e2e_test.go

```diff
@@ -41,67 +42,98 @@ func (s *IntegrationTestSuite) Test01IBCTokenTransfer() {
+	chainBNode.BankSend("10000000000000000000gamm/pool/1", chainB.NodeConfigs[0].PublicAddress, lockupWalletAddrB)
+	chainBNode.BankSend("10000000000000000000gamm/pool/1", chainB.NodeConfigs[0].PublicAddress, lockupWalletSuperfluidAddrB)
+
+	// ensure we can add to new locks and superfluid locks on chainB
+	chainB.LockAndAddToExistingLock(sdk.NewInt(1000000000000000000), "gamm/pool/1", lockupWalletAddrB, lockupWalletSuperfluidAddrB)
+}
+
+// Test04SuperfluidVoting tests that superfluid voting is functioning as expected.
 // It does so by doing the following:
 //- creating a pool
-// - attempting to submit a proposal to enable superfluid voring in that pool
+// - attempting to submit a proposal to enable superfluid voting in that pool
 // - voting yes on the proposal from the validator wallet
 // - voting no on the proposal from the delegator wallet
 // - ensuring that delegator's wallet overwrites the validator's vote
 // This test depends on pool creation to function correctly.
-// As a result, it is prefixed by 03 to run after Test02CreatePool.
-func (s *IntegrationTestSuite) Test03SuperfluidVoting() {
-	const walletName = "superfluid-wallet"
-
-	chain := s.configurer.GetChainConfig(0)
-	node, err := chain.GetDefaultNode()
+// As a result, it is prefixed by 04 to run after Test02CreatePool.
+func (s *IntegrationTestSuite) Test04SuperfluidVoting() {
+	chainA := s.configurer.GetChainConfig(0)
+	chainANode, err := chainA.GetDefaultNode()
 	s.NoError(err)
 
-	// enable superfluid via proposal.
-	node.SubmitSuperfluidProposal("gamm/pool/1", sdk.NewCoin(appparams.BaseCoinUnit, sdk.NewInt(config.InitialMinDeposit)))
-	chain.LatestProposalNumber += 1
-	node.DepositProposal(chain.LatestProposalNumber, false)
-	for _, node := range chain.NodeConfigs {
-		node.VoteYesProposal(initialization.ValidatorWalletName, chain.LatestProposalNumber)
-	}
-
-	walletAddr := node.CreateWallet(walletName)
-	// send gamm tokens to node's other wallet (non self-delegation wallet)
-	node.BankSend("100000000000000000000gamm/pool/1", chain.NodeConfigs[0].PublicAddress, walletAddr)
-	// lock tokens from node 0 on chain A
-	node.LockTokens("100000000000000000000gamm/pool/1", "240s", walletName)
-	chain.LatestLockNumber += 1
-	// superfluid delegate from non self-delegation wallet to validator 1 on chain.
-	node.SuperfluidDelegate(chain.LatestLockNumber, chain.NodeConfigs[1].OperatorAddress, walletName)
-
 	// create a text prop, deposit and vote yes
-	node.SubmitTextProposal("superfluid vote overwrite test", sdk.NewCoin(appparams.BaseCoinUnit, sdk.NewInt(config.InitialMinDeposit)), false)
-	chain.LatestProposalNumber += 1
-	node.DepositProposal(chain.LatestProposalNumber, false)
-	for _, node := range chain.NodeConfigs {
-		node.VoteYesProposal(initialization.ValidatorWalletName, chain.LatestProposalNumber)
+	chainANode.SubmitTextProposal("superfluid vote overwrite test", sdk.NewCoin(appparams.BaseCoinUnit, sdk.NewInt(config.InitialMinDeposit)), false)
```

Comment:
nit: I don't really understand the value of the rename since we are only using one chain in this test

Reply:
I personally like to know which chain's state it is I am modifying. I would agree that this is unnecessary if the state was reset after each run, but since it is retained I think it is helpful. 

Another reason for this is for those learning the layout of the e2e for the first time. It's not obvious that `s.configurer.GetChainConfig(0)` is actually chainA. By referring to it as chainA even in isolation, it is easier to understand as a whole imo. 

If you feel strongly against it though I can change this back!

Reply:
No that makes sense, ty!